src/js/tabs/osd.js

```diff
@@ -1805,6 +1820,7 @@ OSD.chooseFields = function() {
         if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_42)) {
             OSD.constants.STATISTIC_FIELDS = OSD.constants.STATISTIC_FIELDS.concat([
                 F.STAT_TOTAL_FLIGHTS,
+                F.STAT_TOTAL_PACKS,
```

Comment:
It should be added to API_VERSION_1_44, the existing code seems wrong, @limonspb why you added Flight count in 1.42?